docs/source/index.mdx

```diff
@@ -193,4 +183,12 @@ query GetCurrentUserReviews {
 }
 ```
 
-Next, learn how to [define an implementing service](./implementing-services/).
+### Managed federation
+
+In addition to running a gateway with a static list of services, Apollo Gateway can operate in **managed federation** mode, where [Apollo Studio](https://www.apollographql.com/docs/studio/) acts as the source of truth for each subgraph's schema.
+
+This mode enables multiple teams working on a data graph to coordinate when and how underlying subgraphs change. It's recommended for all federated graphs. For more information, read [Managed federation overview](https://www.apollographql.com/docs/studio/managed-federation/overview/).
+
+---
+
+Next, learn how to [define a subgraph](./implementing-services/).
```

Comment:
I _feel_ like you've changed this to ./subgraphs/?
```suggestion
Next, learn how to [define a subgraph](./subgraphs/).
```

Reply:
Oops I sure did! Thanks 😄